README.md

```diff
@@ -174,3 +169,5 @@ defaults write org.raspberrypi.Imager.plist telemetry -bool NO
 When using the app, press <kbd>CTRL</kbd> + <kbd>ALT</kbd> + <kbd>X</kbd> to reveal the **Advanced options** dialog.
 
 In here, you can specify several things you would otherwise set in the boot configuration files. For example, you can enable SSH, set the Wi-Fi login, and specify your locale settings for the system image.
+
+**Note:** The advanced options only affect Raspberry Pi OS; support for other operating systems may be added in the future.
```

Comment:
I wonder if this might need to be reworded to clarify that this is related to **flashing** Raspberry Pi OS (regardless of what OS you're currently running RPi Imager on); rather than inadvertently implying that you can only use the "advanced options" if you're running RPi Imager on Raspberry Pi OS itself? :wink: :upside_down_face:

Reply:
Good point. I've clarified the wording.